sdk/samples/iot/paho_iot_pnp_sample_common.c

```diff
@@ -378,14 +378,14 @@ static void handle_device_property_message(
       break;
 
     // An update to the desired properties with the properties as a payload.
-    case AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_DESIRED_PROPERTIES:
+    case AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_WRITEABLE:
       IOT_SAMPLE_LOG("Message Type: Desired Properties");
       process_device_property_message(message_span, property_response->response_type);
       break;
 
     // A response from a reported properties publish message.
-    case AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_REPORTED_PROPERTIES:
-      IOT_SAMPLE_LOG("Message Type: Reported Properties");
+    case AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_ACKNOWLEDGEMENT:
+      IOT_SAMPLE_LOG("Message Type: Previous property update from device acknowledged by IoT Hub");
```

Comment:
This is a little hard to read. :/

Reply:
It's tricky to capture all this in a single comment since this is async, but i've moved the primary actor here (IoT Hub) to be the subject of sentence rather than at end since I think that makes it a bit clearer.